meas_std_val: 0.03
u_std: [0.007, 0.007, 0.007, 0.007]
initial_range: [1.0, 1.0, 1.0]
num_particles: 50
stale_filter_time: 1.0

max_objects:
  power_cell: 10

labels:
- BACKGROUND
- power_cell
